plugins/inventory/gql_inventory.py

```diff
@@ -63,10 +65,9 @@
       group_by:
           required: False
           description:
-            - List of group names to group the hosts
+            - List of data paths to group by.
```

Comment:
```suggestion
            - List of dot-sparated paths to index graphql query results (e.g. `platform.slug`)
            - The final value returned by each path is used to derive group names and then group the devices into these groups.
            - Valid group names must be string, so indexing the dotted path should return a string (i.e. `platform.slug` instead of `platform`)
            - If value returned by the defined path is a dictionary, 
            an attempt will first be made to access the `name` field, and then the `slug` field. 
            (i.e. `platform` would attempt to lookup `platform.name`, and if that data was not returned, it would then try `platform.slug`)
```